drivers/Cli/command_line_interface.py

```diff
@@ -25,24 +22,19 @@ def repl(userIO, acc, transfer_use_case):
                 userIO.print(f"{e.value} is a non-positive value to transfer.")
 
 
-def main(accounts_repository, transfer_use_case, userIO):
+def main(userIO, login_use_case, transfer_use_case, open_account_use_case):
```

Comment:
type the arguments of main.

Reply:
That make me think about implementing a new Interface Class for UseCases, such as businessUseCase or TransactionalUseCase and do all the UseCases implements it. What do you think?

Reply:
Each Use case receive a different set of parameters, how this interface could be? I don't think that is possible

Reply:
Yeah. Thinking better, there is no way to reproduce lots of UseCases using a interface. The main typing could get wrong.

Reply:
Should I type the repl arguments as well?